dev-infrastructure/docs/development-setup.md

```diff
@@ -259,6 +265,13 @@ Option 2: You can follow the below manual steps from the root of the CS repo on
 # Setup the development.yml
 cp ./configs/development.yml .
 
+# Setup the azure-runtime-config.json
+cp ./configs/azure/example-config.json ./azure-runtime-config.json
```

Comment:
This works for now I guess but in the near future we will need to indicate exactly what content is the desired one